packages/hardhat/test/Challenge1.ts

```diff
@@ -88,23 +92,24 @@ describe("🚩 Challenge 1: 🥩 Decentralized Staking App", function () {
         });
 
         it("Should redeploy Staker, stake, not get enough, and withdraw", async function () {
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
           const [owner, secondAccount] = await ethers.getSigners();
 
           const ExampleExternalContract = await ethers.getContractFactory("ExampleExternalContract");
           exampleExternalContract = await ExampleExternalContract.deploy();
 
           const Staker = await ethers.getContractFactory("Staker");
-          stakerContract = await Staker.deploy(exampleExternalContract.address);
+          stakerContract = await Staker.deploy(exampleExternalContractAddress);
```

Comment:
I was having issues passing the test in this point, was throwing always that it was already completed. Think the problem may be we are passing an old completed address as argument.

Fixed for me adding this before, to refresh with the right address:
`exampleExternalContractAddress = await exampleExternalContract.getAddress();`

My approach may be totally wrong, sorry if so!

Reply:
I don't know, it works for me 🤷‍♂️ . Changed it, also changed waiting time to 72 hours instead of one hour, since readme says to make it 72 hours. Without it tests also fails, didn't noticed it because I tested with 72 seconds. Thanks!

Reply:
Now they're working great for me! 🙌 I had same issue with time, testing with low timeframes, nice catch!